### ISSUE TITLE
Mobile app login does not work if TOTP is enabled

### ISSUE THREAD

**Issue by dazoot** [RocketChat/Rocket.Chat#33723]
### Description:

Logging in on the mobile app (iOS and Android) returns auth failure: Credential were rejected.... if i have TOTP enabled.

### Steps to reproduce:

1. Try to login on the mobile app
2. Enter correct user / pass (works in browser)
3. Click Login

### Expected behavior:

Login to work.

### Actual behavior:

Login does not work. 

I get an error logging IN.

![image](https://github.com/user-attachments/assets/c29c2b07-af82-481b-938e-c1e3578ce962)

### Server Setup Information:

- Version of Rocket.Chat Server: 6.13.0
- License Type: Starter
- Number of Users: 10
- Operating System: Debian
- Deployment Method: tar
- Number of Running Instances: 1
- DB Replicaset Oplog: (oplog Enabled)
- NodeJS Version: v14.21.3
- MongoDB Version: 4.4.29

- Version of Rocket.Chat Server: 
- License Type:
- Number of Users: 
- Operating System: 
- Deployment Method: 
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version:

### Client Setup Information

- iOS Application version 4.53.0.66078
- Operating System: iOS

### Additional context

Login on mobile works after i disable from the browser TOTP 2step authentification.

**Comment by Gummikavalier** [comment]
I cannot reproduce this error with Oauth provisioned RC 6.13.0 system. TOTP enabled.

With LDAP enabled RC I get that error immediately when I arrive on the login page, already before typing in the username and password. However when I bypass that error and type them in, I can get into RC as normal. Also TOTP query follows as it should.

**Comment by reetp** [comment]
I'll refer this to the team for comment.

**Comment by casalsgh** [comment]
Sending to our Mobile experts. thanks for reporting

**Comment by diegolmello** [comment]
@dazoot @Gummikavalier Can you try on 4.54.1?
I'm not sure I understand how to reproduce that, but I can say TOTP is working fine here without LDAP.

If you're on 6.13, create an account on https://open.rocket.chat, which is on v7 atm, and try there as well.
Thanks!

**Comment by Gummikavalier** [comment]
@diegolmello I tested and reproduced the issue now with app version 4.54.1 too.

But in my case I have two other RCs configured into the Rocket.Chat app too, I now cleaned up cache for all of them and logged out from all.

After above I did not encounter the issue anymore.

Also there was no issue anymore when adding the second and third instances back to the app.

I had done cleaning up of cache earlier too, so most probably what fixed the problem was removing other instances too from the app.
